include/openssl/ct.h

```diff
@@ -472,7 +476,7 @@ SCT *o2i_SCT(SCT **psct, const unsigned char **in, size_t len);
 * If |out| points to an allocated string, the signature will be written to it.
 * The length of the signature in TLS format will be returned.
 */
-int i2o_SCT_signature(const SCT *sct, unsigned char **out);
+__owur int i2o_SCT_signature(const SCT *sct, unsigned char **out);
```

Comment:
It is potentially useful to be able to encode/decode the signature separately, as it can appear encoded independently of the rest of the SCT (e.g. in the JSON returned from a log, the signature is encoded in the format that these functions use, and then base64-encoded).